packages/api/deploy/README.md

```diff
@@ -31,7 +31,7 @@ docker run --rm -ti --init \
    -p 8007:80 \
    -e "GRAPHQL_PORT=80" \
    -e "NODE_ENV=development" \
-   -e "ELASTICSEARCH_URL=elastic:9200" \
+   -e "ELASTICSEARCH_URL=localhost:8001/api/v1/namespaces/default/services/elasticsearch:9200/proxy" \
```

Comment:
This would be pointed at the container's port 8001. It should be the host's (`host.docker.internal:8001/api/...`).